TMS.Net07.Homework.DaysOfWeek/hard.cs

```diff
@@ -0,0 +1,314 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.DaysOfWeek
+{
+    class hard
+    {
+        public static void task()
+        {
+            while (true)
+            {
+                int day;
+                int month;
+                int year;
+                Console.WriteLine("Введите вашу дату: ");
+                var partsOfDate = Console.ReadLine().Split('.');
+                if (partsOfDate.Length == 3)
+                {
+                    //string[] partsOfDate = input.Split('.');
+                    bool isRightDay = int.TryParse(partsOfDate[0], out day);
+                    bool isRightMonth = int.TryParse(partsOfDate[1], out month);
+                    bool isRightYear = int.TryParse(partsOfDate[2], out year);
+                    if(isRightDay==true &&isRightMonth==true&&isRightYear==true)
+                    {
+                        Date date = new Date(int.Parse(partsOfDate[0]), int.Parse(partsOfDate[1]), int.Parse(partsOfDate[2]));
+                        date.Print();
+                        date.DayOfWeek();
+                    }
+                    else
+                    {
+                        Console.WriteLine("Неверный ввод");
+                    }
+                }
+                else if (partsOfDate[0].ToUpper() == "EXIT")
+                {
+                    return;
+                }
+                else
+                {
+                    Console.WriteLine("Неверный ввод");
+                }
+
+            }
+        }
+    }
+    enum DaysOfWeek
+    {
+        ПОНЕДЕЛЬНИК,
+        ВТОРНИК,
+        СРЕДА,
+        ЧЕТВЕРГ,
+        ПЯТНИЦА,
+        СУББОТА,
+        ВОСКРЕСЕНЬЕ
+    }
+    class Date
+    {
+        private int m_day;
```

Comment:
```suggestion
        private int _day;
```
Oh, my lovely c++ :)
In C# we usually name private fields with an underscore

Reply:
done))